webpages/settings/style.css

```diff
@@ -51,8 +51,8 @@ h1 {
   align-items: stretch;
 }
 
-.addon-group {
-  margin-bottom: 20px;
+.addon-group:not(:first-child) {
+  margin-top: 20px;
```

Comment:
This will not work properly, because a group header might be hidden with `display:none`. We'd want to target the first visible child, not the first child.

Reply:
Lol there are so many bad ways to go about this and I don't know which one to start with

Reply:
Is hiding the element by removing it instead of `display: none` also a bad way?

Reply:
Solved